bundles/admin/admin-layereditor/view/AdminLayerForm/GeneralTabPane.jsx

```diff
@@ -42,10 +63,18 @@ const GeneralTabPane = (props) => {
                 <TextInput type='text' value={layer.name} onChange={(evt) => controller.setLayerName(evt.target.value)} />
             </StyledComponent>
             <StyledComponentGroup>
-                <LocalizedLayerInfo layer={layer} lang={lang} controller={controller} />
-                <StyledComponent>
-                    <OtherLanguagesPane layer={layer} lang={lang} controller={controller} />
-                </StyledComponent>
+                <LocalizationComponent
+                    labels={localized.labels}
+                    value={localized.values}
+                    languages={Oskari.getSupportedLanguages()}
+                    onChange={controller.setLocalizedNames}
+                    LabelComponent={PaddedLabel}
+                >
+                    <TextInput type='text' name='name'/>
+                    <Padding/>
```

Comment:
<Padding /> is used because inputs inside LocalizationComponent need to be at the "root level"? So we can't just have a StyledTextInput with the padding? There should probably be a comment about this if that is the reason. Otherwise it will get "fixed"/broken in near future.

Reply:
Yes, that's why. And we could make a custom TextInput, but then the style should have padding settings with *!important* since ant-design overrides it otherwise with 0.